src/main/java/org/traccar/model/Device.java

```diff
@@ -149,4 +151,13 @@ public void setDisabled(boolean disabled) {
         this.disabled = disabled;
     }
 
+    @QueryIgnore
+    public boolean getAllowForward() {
```

Comment:
Please don't add this to the device model.

Reply:
Removed it.